homeassistant/components/wyoming/__init__.py

```diff
@@ -0,0 +1,45 @@
+"""The Wyoming integration."""
+from __future__ import annotations
+
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.discovery import async_load_platform
```

Comment:
We shouldn’t use this. We should forward config entry setup

Reply:
I'm forwarding now, but after loading the Wyoming info from the endpoint. This is to avoid loading it multiple times, once in each platform.

Reply:
Moved the info loading into the config flow.